src/firecracker/src/main.rs

```diff
@@ -36,7 +36,10 @@ use vmm::signal_handler::register_signal_handlers;
 use vmm::vmm_config::instance_info::{InstanceInfo, InstanceState};
 use vmm::{EventLoopExitReason, Vmm};
 
-const DEFAULT_API_SOCK_PATH: &str = "/tmp/firecracker.socket";
+// The reason we place default firecracker.socket under /run is that firecracker.socket is a
+// runtime variable.
+// see https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch03s15.html for more information
+const DEFAULT_API_SOCK_PATH: &str = "/run/firecracker.socket";
```

Comment:
We also need a changelog entry for this because it is an user facing change.

Reply:
added changelog 👍